python-coding-guidelines.md

```diff
@@ -82,10 +92,29 @@ dark corners of cyclomatic complexity, for nested loops you might want to consid
 function or method. While the code will get a little more verbose, and you might face other difficulties like having to pass
 around variables or even promoting some to instance variables, the resulting code is still much simpler to read then
 condensed web of loops mutating maybe tens of local variables.
- 
+
+Instead of writing loops with a single condition inside be pythonic and consider using list comprehensions.
```

Comment:
I'd put this someplace else because this is basically preferential, i.e. there's nothing wrong with the original code besides being a little bit more verbose. Tangentially, I'd like to see these things refactored into functions with decent name and docstring, while the functionality itself might be really simple, it doesn't answer the critical question as to why the piece of code is there, and more importantly it's much easier to write tests for these.

For contrast:

```python
# code
res = [s for s in time_sources if s not in ["42", "22"]]
# code
```

with:

```python
def filter_biffsquiggled_time_sources(time_sources):
   ''' Biffsquiggled time sources have their value either 42, 22 because that is the value 
       described by IETF in RFC666 regarding time sources allocations
   '''
   return [s for s in time_sources if s not in ["42", "22"]]
```

Reply:
You are right, that's more about preference. Removed the passage